dev/test/specs/starter-test.js

```diff
@@ -153,7 +162,7 @@ function waitForStillness() {
       }
       return matches > desiredMatches;
     },
-    10000,
+    20000,
```

Comment:
@smalls is doubling this here ok?

Reply:
I tried to add some clarifying comments to the commit message, but they don't show well here (unfortunately). I had to increase the timeout so tests would pass. the system seems to be taking longer to settle.

The increase below (to 60006) is to expand the overall time so the 20k here doesn't cause the whole test to timeout.

Reply:
Good question, though, and I'm not sure how to explain it better in the code. Ideally we shouldn't have to increase these timeouts, and at some point we'll need to start reducing them (maybe?)

Reply:
Another option would be to do what you mentioned above, to add a comment about when these can be changed?